src/index.html

```diff
@@ -11,7 +11,140 @@
+     method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="form-filde">
+      Surname:
+          <input type="text" name="Surname">
+        </label>
+
+        <label class="form-filde">
+        Name:
+          <input type="text"name="Name">
+        </label>
+
+        <label class="form-filde">
+        How old are You?
+          <input type="number"
+          name="old"
+          value="12"
+          min="1"
+          max="100"
+          >
+        </label>
+
+        <label class="form-filde">
+        Full date of birth:
+          <input type="date"
+          name="date of birth"
+          placeholder="дд.мм.гггг"
+          >
+        </label>
+
+        <label class="form-fild">
+        I accept the term of the agreement
+          <input type="checkbox"
+          name="accept"value="accept"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="form-filde">
+          E-mail:
+          <input type="email"
+          name="email"
+          placeholder="email@example.com"
+          required
+          >
+        </label>
+
+        <label class="form-fild">
+          Password:
+          <input type="password"
+          name="Password"
+          required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you</legend>
+        <label class="form-filde">
+          Do you love cat?
+          <input type="radio" id="Lovecat"
+          name="quest" value="question"
+          >
+          <label for="Lovecat">Yes</label>
+
+          <input type="radio" id="NoLovecat"
+          name="quest" value="question"
+          >
+          <label for="NoLovecat">No</label>
+        </label>
+
+        <lebel class="form-filde">
+          What is your favorite color?
+          <input type="color" name="color">
+        </lebel>
+
+        <label class="form-filde">
+          What time do you go to bed?
+          <input type="time" name="bedtime">
+        </label>
+
+        <label class="form-filde">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="Fiat">Fiat</option>
+            <option value="Tesla">Tesla</option>
+            <option value="Lanos">Lanos</option>
+          </select>
+        </label>
+
+        <label class="form-fild"
+        >
+          How do you rate our work?
+          <input type="range"
+          name="rate"
+          min="0" max="10"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label class="form-filde"
+        >
+          Comments:
+          <textarea name="comments"
+          autocomplete="off"
+          rows="2"
+          cols="20">
+          Enter your Comments...
+          </textarea>
```

Comment:
```suggestion
          <textarea 
            name="comments"
            autocomplete="off"
            rows="2"
            cols="20"
          ></textarea>
```
Let's fix this problem
![image](https://user-images.githubusercontent.com/96409563/214004171-a6345fd7-6ae8-40d2-bf4c-651e477c82b2.png)